backend/game/engine/player_queue.py

```diff
@@ -10,7 +10,7 @@ def __init__(self):
         self.current_player = None
         self.player_cycle = []
 
-    def _add_player_to_game(self, new_player):
```

Comment:
@gpfaia I'm not seeing why this was a protected method. If this should be protected let me know and we can change it back

Reply:
No originally just wanted people to be mindful of adding players after the game was started was the original intention